### ISSUE TITLE
Add various mappings to a concordance in one action

### ISSUE THREAD

**Issue by guitarster** [gbv/cocoda#686]
Currently it is only possible to add one mapping to a concordance at the same time. 
It would be very helpful, if it would be possible to add various existing mappings to a concordance at the same time. 
A workflow could be: first searching for certain mappings in the search-tab (optionally use the filter); then select all the mappings and put them in one action to a concordance

**Comment by nichtich** [comment]
How do you envision "select all the mappings"? Just the full result set? This would be easiest to implement.

**Comment by guitarster** [comment]
Yes this is what i thougt: the full result set.

**Comment by stefandesu** [comment]
My question is: How often will you actually use this? All future mappings will likely be saved directly into the concordance, so this would only be to deal with already existing mappings, and once they are dealt with, it would barely be used anymore.

If I remember correctly, I had already suggested a feature like this in #623, but we decided to perform this manually in the backend for now, unless there's a use case where this would be used regularly.
